### PR TITLE
Update quest.yml

### PR DESCRIPTION
**Pull request by IEvangelist**
Update quest trigger events.

### CODE REVIEW

File: .github/workflows/quest.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: "quest import"
 on:
   issues:
     types:
-      [ labeled ]
+      [ labeled, closed, reopened, assigned, unassigned ]
```
Comment (BillWagner) [inline .github/workflows/quest.yml:5]: With these additional events, the logic currently on L18 needs to be updated.

Comment (IEvangelist) [reply .github/workflows/quest.yml:5]: I'm not sure how, could you tell me what the rules are for that scenario? I thought the idea would be to only run when there is a trigger label present regardless.

Comment (IEvangelist) [reply .github/workflows/quest.yml:5]: So, add the inquest label as part of the condition too - right?

Comment (IEvangelist) [reply .github/workflows/quest.yml:5]: Updated

**Review by BillWagner — APPROVED** [review]
This LGTM. Let's :shipit:

**Pull request merged**